### DISCUSSION TITLE
CRLF's in .md file do not result in linebreaks in pdf

### DISCUSSION THREAD

**Discussion by marcelslats-g4s** [rfennell/AzurePipelines#1300] [Q&A] [answered]
I'm trying the Azure DevOps.Wiki PDF Export extension.
I generate a markdown with 'Generate Release Notes' and then I like to export the page to pdf.

The following is rendered okay by 'Generate Release Notes' to an .md with crlf at the end of the lines but when the pdf is generated they are not on new lines.
So it looks like the crlf is not respected.
Am I doing something wrong here?

**Release Number**: {{releaseDetails.name}}
**Release completed**: {{releaseDetails.modifiedOn}}     
**Build Number**: {{buildDetails.id}}
**Compared Release Number**: {{compareReleaseDetails.name}}    
**Build Trigger PR Number**: {{lookup buildDetails.triggerInfo 'pr.number'}}

**Comment by rfennell** [comment, marked as answer]
One question first, what agent are you running the task on, Windows, Linux or Mac? Does swapping the agent type have an effect?

All my PDF export task does is act as a wrapper for [MaxMelcher/AzureDevOps.WikiPDFExport](https://github.com/MaxMelcher/AzureDevOps.WikiPDFExport) command line tool. As you can run this tool as a command line, against the .MD file my task crates, you can make sure it is the tool that has the issue and not my wrapper task. 

If you get the same behaviour with the tool, and I expect you will, best option then is to log a call on [MaxMelcher/AzureDevOps.WikiPDFExport](https://github.com/MaxMelcher/AzureDevOps.WikiPDFExport) Issues page

**Reply by marcelslats-g4s** [reply]
Hi Richard.,

This is on a windows agent.
I downloaded the tool and ran it in debug to see what HTML was generated and found out that the text is wrapped in a `<p>` and only when I use 2 spaces in front of the CRLF it adds a `<br />`.
I posted a new issue on the issues page of Max Melcher for this.

Thank you.
